src/base/projects/View.svelte

```diff
@@ -74,6 +76,31 @@
     if (path !== newParams.path) path = newParams.path;
   }
 
+  const GetAllAnchors = `
+    query GetAllAnchors($project: Bytes!, $org: ID!) {
+      anchors(orderBy: timestamp, orderDirection: desc, where: { objectId: $project, org: $org }) {
+        multihash
+        timestamp
+      }
+    }
+  `;
+
+  interface AnchorObject {
+    timestamp: number;
+    multihash: string;
+  }
+
+  async function getAllAnchors(urn: string, anchors?: string | null): Promise<string[]> {
+    if (! anchors) {
+      return [];
+    }
+    const unpadded = decodeRadicleId(urn);
+    const id = ethers.utils.hexZeroPad(unpadded, 32);
+    const allAnchors = await querySubgraph(config.orgs.subgraph, GetAllAnchors, { project: id, org: anchors });
+    return allAnchors.anchors
+      .map((anchor: AnchorObject) => formatProjectHash(ethers.utils.arrayify(anchor.multihash)));
+  }
```

Comment:
Can we perhaps move this in a `lib.ts` perhaps? Seems like it shouldn't belong in a view.